src/GridFS/Bucket.php

```diff
@@ -132,6 +133,10 @@ class Bucket
      */
     public function __construct(Manager $manager, string $databaseName, array $options = [])
     {
+        if (isset($options['disableMD5']) && $options['disableMD5'] === false) {
+            trigger_deprecation('mongodb/mongodb', '1.10', 'Setting GridFS "disableMD5" option to "false" is deprecated and will not be supported in version 2.0.');
```

Comment:
"1.10" looks incorrect here. Should it be "1.18", since that will be the version where this change is implemented?

Reply:
Fixed. Copilot need to be smarter with package versions.